### ISSUE TITLE
export command has error output for a file command

### ISSUE THREAD

**Issue by beauraines** [xwmx/nb#293]
When I use the `nb export` command, it outputs an error from the `file` command. I cannot figure out where `file` is being called, but I know it happens after the check for existing file.

```
$ nb export 1124 /tmp/foo.pdf
File exists at /tmp/foo.pdf
Overwrite existing file? [y/N] y
file: invalid option -- o
Usage: file [-bcCdEhikLlNnprsSvzZ0] [--extension] [--mime-encoding]
            [--mime-type] [-e <testname>] [-F <separator>]  [-f <namefile>]
            [-m <magicfiles>] [-M magicfiles] [-P <parameter=value>] [--exclude-quiet]
            <file> ...
       file -C [-m <magicfiles>]
       file [--help]
Try `file --help' for more information.
Exported: [1124] dan_sync_2023-12-26.md "Dan Sync 2023-12-26"
To:        /tmp/foo.pdf
```

This doesn't occur if if use the `export pandoc` subcommand.

```
$ nb export pandoc 1124 -o /tmp/foo.pdf
$
```

So, best I can figure is that it is coming from someplace in the `_export()` function, after [Line 15115](https://github.com/xwmx/nb/blob/51e10443c4caac426e84d6b1967a3a46e8904fa2/nb#L15115C6-L15115C6)

I'm running OS X Sonoma 14.2.1 (23C71) with 

```$ file --version
file-5.41
magic file from /usr/share/file/magic
```

**Comment by beauraines** [comment]
Searching the `nb` script, all the statements calling `file` _seem_ to look okay to me, so I'm still at a loss where this is coming from.

```
$ grep -nE 'file -.*o.*' /usr/local/bin/nb
17213:                  _this_pipe=$(_temp file --touch) && {
17255:                  _this_pipe=$(_temp file --touch) && {
20763:                  _mime_type="$(file -b --mime-type "${_source_path}" 2>/dev/null || :)"
20799:                  _mime_type="$(file -b --mime-type "${_source_path}" 2>/dev/null || :)"
```

**Comment by xwmx** [comment]
@beauraines Thanks for the detailed information. This should be fixed as of version 7.9.1. Let me know if you run into any more issues with it. Thanks!

**Comment by beauraines** [comment]
Thank you! That did it!

And that was fast too.
